### ISSUE TITLE
change pretrain to pretrained?

### ISSUE THREAD

**Issue by CarloLucibello** [FluxML/Metalhead.jl#224]
Should we have 
```julia
VGG(11, pretrained=true)
```
instead of 
```
VGG(11, pretrain=true)
```
The latter seems a command...

**Comment by CarloLucibello** [comment]
Even better, as in torchvision we could have a `weights` argument instead where we can pass more specific values, in case of multiple pertaining options:
```julia
VGG(11, weights="IMAGENET1K_V1")
```

**Comment by darsnack** [comment]
Yes the loading logic already supports that, and the plan was to move to that once we have multiple weight options.
